### PR TITLE
lndmanage: init at 0.11.0

### PR DESCRIPTION
**Pull request by mmilata**
###### Motivation for this change

Utility useful for operators of lnd which is packaged in nixpkgs.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/blockchains/lndmanage.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    substituteInPlace setup.py --replace '==' '>='
+  '';
+
+  preCheck = ''
+    export HOME=$(mktemp -d)
+  '';
+
+  meta = with lib; {
+    description = "Channel management tool for lightning network daemon (LND) operators";
+    homepage = src.meta.homepage;
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/lndmanage.nix:40]: Please set the homepage explicitly.

File: pkgs/applications/blockchains/lndmanage.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    license = licenses.mit;
+    maintainers = with maintainers; [ mmilata ];
+    platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/lndmanage.nix:43]:
```suggestion
```

set by buildPythonApplication to the correct value.

File: pkgs/applications/blockchains/lndmanage.nix (hunk 1 above)
Comment (mmilata) [reply pkgs/applications/blockchains/lndmanage.nix]: Fixed in c9405e3ad6fd6a26db46ed82eb59a401bdbb226b

File: pkgs/applications/blockchains/lndmanage.nix (hunk 2 above)
Comment (mmilata) [reply pkgs/applications/blockchains/lndmanage.nix]: Fixed in c9405e3ad6fd6a26db46ed82eb59a401bdbb226b

**Comment by risicle** [comment]
Builds, runs, macos 10.14

**Review by RaghavSood — APPROVED** [review]

**Comment by mmilata** [comment]
Thank you!

**Pull request merged**